frontend/src/app/main/ui/dashboard/grid.cljs

```diff
@@ -34,7 +34,7 @@
    [cuerdas.core :as str]
    [rumext.v2 :as mf]))
 
-(log/set-level! :info)
+(log/set-level! :warn)
```

Comment:
`:info` was there for properly logging the thumbnails loading, I need it activated once deployed.

Reply:
we need a way to overwrite it externally but right now we don't have this option

Reply:
wtf?

well, if so, we need to leave it in info.